flink-python/src/main/java/org/apache/flink/table/utils/python/PythonTableUtils.java

```diff
@@ -512,7 +512,8 @@ private static Function<Object, Object> converter(
         }
 
         return c -> {
-            if (c.getClass() != byte[].class || dataType instanceof PickledByteArrayTypeInfo) {
+            if (c != null && c.getClass() != byte[].class
```

Comment:
If c== null, we should return c immediately.

Reply:
@HuangXingBo  Thanks for your comment,
you're right,  this is a judgment that the data type is not what we want, I will change it now